File: uri-online-judge/java/P_1169.java
import java.util.Scanner;

/**
 * P_1169
 */
public class P_1169 {

  public static void main(String[] args) {
    Scanner keyboard = new Scanner(System.in);
    int x = keyboard.nextInt();
    int grains = 1, day = 1;
    while (day <= x) {
      grains *= 2;
      day++;
    }
    System.out.println(grains / 12000 + " kg");
    keyboard.close();
  }
}

File: uri-online-judge/c/p_1169.c
#include <stdio.h>

int main()
{
  int numOfTest, x, day, i;
  double grains;
  unsigned long kg;

  scanf("%d", &numOfTest);
  for (i = 0; i < numOfTest; i++)
  {
    scanf("%d", &x);
    grains = 1;
    day = 1;
    while (day <= x)
    {
      grains = grains * 2;
      day = day + 1;
    }
    kg = grains / 12000;
    printf("%lu kg\n", kg);
  }
  return 0;
}